lib/src/components/NxThreatIndicatorLegend/NxThreatIndicatorLegend.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import classnames from 'classnames';
+import {
+  NxThreatIndicatorLegendProps,
+  nxThreatIndicatorLegendPropTypes,
+  ThreatProps
+} from './types';
+import { NxThreatIndicator } from '../..';
+import './NxThreatIndicatorLegend.scss';
+
+const capitalize = (str: string) => str.charAt(0).toUpperCase() + str.slice(1);
+
+const ThreatIndicator = ({ threatType }: ThreatProps) => {
+  return (
+    <div className="threat-container">
```

Comment:
This class name still needs to be prefixed with `nx-`

Reply:
> This class name still needs to be prefixed with `nx-`

The `nx-` prefix has been added
921840723ff0d3a4a4c77d03ffa10d90da865560